src/containers/AudioUploader/components/AudioForm.jsx

```diff
@@ -78,13 +78,9 @@ const rules = {
 };
 
 class AudioForm extends Component {
-  constructor(props) {
-    super(props);
-    this.handleSubmit = this.handleSubmit.bind(this);
-    this.state = {
-      savedToServer: false,
-    };
-  }
+  state = {
```

Comment:
this.state?

Reply:
Kopierte tilsvarende oppsett fra ImageForm.jsx. Funker slik. Muligens det funker fordi det er en Component.

Reply:
Reverter akkurat dette. Ser det fungerer uten denne endringa.